tests/sentry/event_manager/test_event_manager.py

```diff
@@ -2435,6 +2435,29 @@ def test_perf_issue_creation_over_ignored_threshold(self):
             assert data2["hashes"] == []
             assert data3["hashes"] == [expected_hash]
 
+    @override_options(
+        {
+            "performance.issues.slow_span.problem-creation": 1.0,
+            "performance_issue_creation_rate": 1.0,
+        }
+    )
+    @override_settings(SENTRY_PERFORMANCE_ISSUES_REDUCE_NOISE=True)
```

Comment:
This is kind of scary, IMO. Self-hosted Sentry users might not have this set to `False`, and they'd start getting these slow span issues, no?

Reply:
It's overwritten just for the test though. I think issue creation shouldn't be affected

Reply:
Oh good catch. Yeah this is a concerning possibility, I'll add another check just in case to delete and slow DB issue grouphashes

Reply:
@udameli I realize that, I just mean that the fact that this setting is being overridden indicates that it _must_ be `True` for this approach to work

Reply:
I think he means specifically for the actual creation code here: https://github.com/getsentry/sentry/blob/42e83537f89626030acdbdaed34cedc26628917f/src/sentry/event_manager.py#L2301

If that setting is false, the `should_create_group` check won't run and we'd end up creating the slow DB issues

Reply:
I do mean that yes, sorry! I put the comment in the wrong place!

Reply:
ah gotcha!

Reply:
That's a good point. Do we still need this `SENTRY_PERFORMANCE_ISSUES_REDUCE_NOISE` setting at all? It made sense to have this setting when we thought this was temporary, as per the comment:

```python
# temporary fix to limit group creation to grouphashes seen 3+ times in a 24-48 hour period
```

but now it doesn't really seem to be temporary for existing (N+1) or future (slow DB) detectors.

Reply:
yeah it's not going to be temporary. AFAIK `SENTRY_PERFORMANCE_ISSUES_REDUCE_NOISE` can go 👍

Reply:
I can remove it in a future PR